lib/amorail/custom_fields.rb

```diff
@@ -1,3 +1,5 @@
+require 'pry-byebug'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.